docker-compose.yml

```diff
@@ -61,7 +61,7 @@ services:
       - KAFKA_URL=kafka:9092
     command: "bash '/analytics/start-jobmanager.sh'"
     volumes:
-      - ./docker-mounts/flink-jobs:/analytics
+      - ./analytics:/analytics:ro
```

Comment:
`:ro` ??

Reply:
Read only: The Flink container should only get read access to this volume